bundles/best_practices.rst

```diff
@@ -521,7 +521,9 @@ The ``composer.json`` file should include at least the following metadata:
 
 ``autoload``
     This information is used by Symfony to load the classes of the bundle. It's
-    recommended to use the `PSR-4`_ autoload standard.
+    recommended to use the `PSR-4`_ autoload standard. Use the namespace as key,
+    and the location of the bundle's main class (relative to ``composer.json``)
+    as value, example: ``"autoload": { "psr-4": { "JohnSmith\\BlogBundle\\": "" } }``
```

Comment:
adding src/ in the example would be even more descriptive (and is common practice):
`"autoload": { "psr-4": { "JohnSmith\\BlogBundle\\": "src/" } }`